src/shared/components/chevron-button/ChevronButton.test.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import ChevronButton from './ChevronButton';
+
+describe('<ChevronButton />', () => {
+  describe('default', () => {
```

Comment:
no need for the nested describe?

Reply:
> I'm not entirely clear on how the sorting works; but it is similar to searching across the table data in the sense that when not all data is available on the client, and the client requests the data from the server on a per-page basis, changing the sort order should trigger an http request, during which we might want to show some kind of a loading state UI in the table. Can't tell whether the current implementation of the table would support something like this.

As I mentioned during the daily scrum, we could add something like `useServerSideSorting` or pass custom sorting function for individual columns. I can mock up something in this PR if you want.